### PR TITLE
added moodle functionality to calendar

### PR DESCRIPTION
**Pull request by WE-dev1**

### CODE REVIEW

File: components/Calendar/index.js [hunk 1]
```diff
@@ -44,6 +59,53 @@ export default function HomeCalendar({ getCaloriesConsumed, isVisible }) {
     }
   }
 
+  function handleClickDay(date, event) {
+    const unixTileDate = new Date(
+      date.getFullYear(),
+      date.getMonth(),
+      date.getDate()
+    ).getTime();
+    console.log("date:", date);
```
Comment (rabe200) [inline components/Calendar/index.js:68]: no console.logs for the PRs

File: components/Calendar/index.js [hunk 2]
```diff
@@ -75,9 +137,42 @@ export default function HomeCalendar({ getCaloriesConsumed, isVisible }) {
         onClick={(event) => {
           event.target.stopPropagation();
         }}
+        onClickDay={handleClickDay}
         // ---
```
Comment (rabe200) [inline components/Calendar/index.js:141]: same as for the c.logs goes for comments :/

**Review by rabe200 — APPROVED** [review]
lgtm, just get rid of the console.logs and "comments" :)

File: components/Calendar/index.js (hunk 1 above)
Comment (WE-dev1) [reply components/Calendar/index.js:68]: thx for the reminder!

**Pull request merged**